UI/Selenium/Utilities/BrowserType.cs

```diff
@@ -4,6 +4,9 @@ public enum BrowserType
     {
         Firefox,
         Chrome,
+        Edge,
+        IE,
```

Comment:
We do not need to worry about IE much as it's obsolete and I have not had any requests to support it.  If this cannot be made to work then it can be skipped.

Reply:
i will remove it